_install-and-configure/install-opensearch/helm.md

```diff
@@ -56,68 +64,74 @@ The default Helm chart deploys a three-node cluster. We recommend that you have
 
    ```bash
    NAME                            	CHART VERSION	APP VERSION	DESCRIPTION                           
-   opensearch/opensearch           	1.0.7        	1.0.0      	A Helm chart for OpenSearch           
-   opensearch/opensearch-dashboards	1.0.4        	1.0.0      	A Helm chart for OpenSearch Dashboards
+   opensearch/opensearch                  	3.1.0        	3.1.0      	A Helm chart for OpenSearch                      
+   opensearch/opensearch-dashboards       	3.1.0        	3.1.0      	A Helm chart for OpenSearch Dashboards
+   ```
+1. Create minimal values.yaml file:
```

Comment:
```suggestion

1. Create a minimal `values.yaml` file:
```

Reply:
Should we specify the directory for this file?

Reply:
@kolchfa-aws I don't think it necessary, as the path needs to be provided with -f or --values flag